### ISSUE TITLE
Missing css rules in default preons config

### ISSUE THREAD

**Issue by gemmadlou** [preons/preons#35]
```
/* Temporary notes */
      .sticky {
        position: sticky;
      }
      .bxsh-light {
        box-shadow: -1px 5px 15px -5px #6c67a073;
      }
      .bg-off-white {
        background-color: #fbfbff;
      }

      .bca-off-whited {
        border-color: #e4e7f7;
      }

      .visually-hidden {
        position: absolute;
        overflow: hidden;
        clip: rect(0 0 0 0);
        width: 1px;
        height: 1px;
        margin: -1px;
        border: 0;
        padding: 0;
      }
```

**Comment by gemmadlou** [comment]
:rocket: Issue was released in [`v0.4.2`](https://github.com/preons/preons/releases/tag/v0.4.2) :rocket:

**Comment by gemmadlou** [comment]
:rocket: Issue was released in [`v0.4.2`](https://github.com/preons/preons/releases/tag/v0.4.2) :rocket:

**Comment by gemmadlou** [comment]
:rocket: Issue was released in [`v0.4.2`](https://github.com/preons/preons/releases/tag/v0.4.2) :rocket:
